packages/esm-service-queues-app/src/helpers/useQueues.ts

```diff
@@ -1,13 +0,0 @@
-import { openmrsFetch } from '@openmrs/esm-framework';
-import useSWRImmutable from 'swr/immutable';
-import { type QueueServiceInfo } from '../types';
-
-export function useQueues(locationUuid?: string) {
-  const apiUrl = '/ws/rest/v1/queue' + (locationUuid ? `?location=${locationUuid}` : '');
-
-  const { data } = useSWRImmutable<{ data: { results: Array<QueueServiceInfo> } }, Error>(apiUrl, openmrsFetch);
-
-  return {
-    queues: data?.data?.results ?? [],
-  };
-}
```

Comment:
For now, we might want to keep this file in place, maybe add a comment indicating that this is deprecated in favor of the hooks/useQueues function.  This way we don't affect existing functionality while we refine these new approaches.